### PR TITLE
Have tinkerbell follow the regular helm pattern

### PR DESCRIPTION
**Pull request by TerryHowe**
/hold

It looks like this helm chart is pulling from quay. Do we change all the image locations when running this?

### CODE REVIEW

**Comment by abhinavmpandey08** [comment]
> It looks like this helm chart is pulling from quay. Do we change all the image locations when running this?
The quay endpoints don't ever get used. We override all the images from the CLI when installing this chart. 
I will replace the quay endpoints with the `public.ecr.aws` ones in a future PR to avoid any confusion

File: projects/tinkerbell/tinkerbell-chart/helm/sedfile.template [hunk 1]
```diff
@@ -0,0 +1 @@
+s,0.1.3,${IMAGE_TAG},g
```
Comment (abhinavmpandey08) [inline projects/tinkerbell/tinkerbell-chart/helm/sedfile.template:1]: what does this get used for? 
Can we generate this dynamically so we don't have to keep another copy of `GIT_TAG`?

Comment (TerryHowe) [reply projects/tinkerbell/tinkerbell-chart/helm/sedfile.template:1]: The build version is set to ${GIT_TAG}-${BUILD_NUMBER}

Comment (TerryHowe) [reply projects/tinkerbell/tinkerbell-chart/helm/sedfile.template:1]:
```
tlhowe@3c22fb41240e tinkerbell-chart % diff -qr old new
Files old/tinkerbell-chart/Chart.yaml and new/tinkerbell-chart/Chart.yaml differ
Files old/tinkerbell-chart/requires.yaml and new/tinkerbell-chart/requires.yaml differ
tlhowe@3c22fb41240e tinkerbell-chart % diff old/tinkerbell-chart/Chart.yaml new/tinkerbell-chart/Chart.yaml 
6c6
< version: 0.1.3
---
> version: 0.1.3-c7c82c3bfc0dd37a3f30adc7cc09feafb2c72980
tlhowe@3c22fb41240e tinkerbell-chart % diff old/tinkerbell-chart/requires.yaml new/tinkerbell-chart/requires.yaml 
4c4
<   name: tinkerbell/tinkerbell-chart-0.1.3
---
>   name: tinkerbell/tinkerbell-chart-0.1.3-c7c82c3bfc0dd37a3f30adc7cc09feafb2c72980
tlhowe@3c22fb41240e tinkerbell-chart %
```

Comment (TerryHowe) [reply projects/tinkerbell/tinkerbell-chart/helm/sedfile.template:1]: for dev builds, you get the github sha, but for prod builds, the code build build number

Comment (abhinavmpandey08) [reply projects/tinkerbell/tinkerbell-chart/helm/sedfile.template:1]: awesome, let's merge this then!

**Pull request merged**